viv_utils/emulator_drivers.py

```diff
@@ -390,13 +393,22 @@ def _runFunction(self, funcva, stopva=None, maxhit=None, maxloop=None, strictops
                 if startpc == stopva:
                     return
 
-                # Check straight hit count...
-                if maxhit != None:
-                    h = hits.get(startpc, 0)
-                    h += 1
-                    if h > maxhit:
-                        break
-                    hits[startpc] = h
+                if op and (op.prefixes & PREFIX_REP):
```

Comment:
so on the first iteration `op` is not initialized? maybe we could move this check below, or move the parsing of `op` up here?